models/subscription.go

```diff
@@ -0,0 +1,38 @@
+package models
+
+import (
+	"github.com/fatih/structs"
+	"time"
+)
```

Comment:
```
import (
	"time"
	
	"github.com/fatih/structs"
)
```

imports should be structured in following groups
imports (
            // go pkgs 

            // github libraries 

            // within project imports
)